### PR TITLE
Add more documentation about implementation and background of Importer

### PR DESCRIPTION
**Pull request by rytswd**
Fixed

- Missing upper bound line range is correctly handled

Added

- Background and implementation details of Importer
- Roadmap
- Annotation details

Updated

- Use of Export Marker for README.md in this repo

### CODE REVIEW

**Comment by rytswd** [comment]
I'll be adding more documentation around the background and implementation details, as well as more detailed list of annotations, and current limitations. These are the must have for v0.1.0 stable release.

File: internal/parse/convert.go [hunk 1]
```diff
@@ -146,6 +147,8 @@ func processTargetDetail(annotation *file.Annotation, input string) error {
 				return fmt.Errorf("%w, %v", ErrInvalidSyntax, err)
 			}
 			annotation.TargetLineTo = upperBound
+		} else {
```
Comment (sryoya) [inline internal/parse/convert.go:150]: Why don't you do like this? Will there be any other conditions?

```.go
annotation.TargetLineTo = math.MaxInt32 // TODO: Consider making this Int64
if ub != "" {
// ....
}
```

**Review by sryoya — APPROVED** [review]
LGTM with a minor question!

File: internal/parse/convert.go (hunk 1 above)
Comment (rytswd) [reply internal/parse/convert.go]: 100% agreed, I was mad or something....

fa4a970

**Review by sryoya — APPROVED** [review]

**Pull request merged**